src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Listing/Listing.html

```diff
@@ -34,6 +34,7 @@ <h3 class="sort-header">{{ "TR__SORT_BY" | translate }}</h3>
             </a>
         </div>
     </div>
+    <div class="listing-counter" data-ng-if="counter && elements.length > 0">{{elements.length}} {{contentName | translate}}</div>
```

Comment:
because in that case, the `listing-empty-text` tells the user that the listing is empty. I deemed putting the counter on top of that redundant.